pyroSAR/drivers.py

```diff
@@ -1173,6 +1173,173 @@ def getCorners(self):
         return self.meta['corners']
 
 
+class EORC_PSR(ID):
+    """
+    Handler class for ALOS-2/PALSAR-2 data in EORC (Earth Observation Research Center) Path format
+    
+    Sensors:
+        * PALSAR-2
+
+    PALSAR-2:
+        Reference: 
+            ALOS-2/PALSAR-2 EORC PAth Product Format Description (First Edition, [access restricted])
+            (`JAXA 2016 <PALSAR2_25m_PATH_product_format_NDX-150019_ENG.pdf>`_).
```

Comment:
Just write the following:
`NDX-150019: ALOS-2/PALSAR-2 EORC Path Product Format Description (JAXA 2016)`